docs/builtins/General/enforce-verifier.md

```diff
@@ -24,14 +24,66 @@ The `enforce-verifier` function returns a boolean value indicating whether the s
 
 ### Examples
```

Comment:
I feel like we should highlight that `verifier plugins` are implemented in chainweb. We currently *only* provide `hyperlane_v3_message` (see https://github.com/kadena-io/chainweb-node/blob/e0acda06fcda6217e2241b728f266199b39bee42/src/Chainweb/Version/Mainnet.hs#L222C55-L222C75)

The example sections always assume a particular verifier to be implemented, in this new case the `signed_list`.
Within the pact repl, all verifiers can be simulated using the `env-verifiers` builtin - but they are not available on-chain (mainnet, devnet, ...).

Maybe @edmundnoble can comment here. My general understanding is that verifier plugins are a light-weight addition we can make to (mainly) partners to provide more complex integrations\infrastructure.

Reply:
Added a pointer to the KIP